falcon/testing/client.py

```diff
@@ -347,6 +347,13 @@ def json(self) -> Optional[Union[dict, list, str, int, float, bool]]:
 
         return json_module.loads(self.text)
 
+    def __repr__(self):
+        content_type = ''
+        if 'Content-Type' in self.headers.keys():
+            content_type = self.headers['Content-Type']
+
+        return 'Result<{} {} {}>'.format(self.status, content_type, self.content[:20])
```

Comment:
So if `content_type == ''`, would you get two spaces between status and content?

Reply:
Ahh, sorry about that.
Will correct in the next commit.

Reply:
First way that came to my mind to handle extra spaces is this:
```
    def __repr__(self):
        content_type = self.headers.get('Content-Type', '')
        repr_result = 'Result<{} {} {}>'.format(
            self.status, content_type, self.content[:20]
        )

        return " ".join(repr_result.split())
```
And i have a feeling this isn't the best way to go about it.

Or i can change the separator instead of spaces, to colon or maybe a comma.
Like this `Result<200 OK::b'hello'>`
What do you think?

Reply:
Changed the syntax to `Result<Status, Content-Type, Content>`.
Where now if no content type header was included, `__repr__` will give a prettier result imo.